### ISSUE TITLE
Utils script does not load

### ISSUE THREAD

**Issue by janmarek** [patw0929/react-intl-tel-input#72]
Hi,
I tried to use this library in my project but it does not load utils script. There is no request in chrome developer tools. Do you have any idea what could cause it?

```js
import IntlTelInput from 'react-intl-tel-input';

export default class PhoneInput extends React.Component {

    // ...

    render() {
        return <div className="form-group">
            {this.props.label && <label className={classnames('control-label', this.props.labelClassName)}>
                {this.props.label}
            </label>}
            <IntlTelInput
                onPhoneNumberChange={this.handleChange}
                css={['intl-tel-input', 'form-control']}
                value={this.props.value}
                defaultCountry={this.props.initialCountry ? this.props.initialCountry.toLowerCase() : ''}
                utilsScript="/assets/libphonenumber.js"
            />
        </div>;

        // I can put anything to utilsScript, the result is still the same
    }

}
```

**Comment by patw0929** [comment]
Do you import the `libphonenumber.js` manually like usage in the example?

`import 'file?name=libphonenumber.js!./node_modules/react-intl-tel-input/dist/libphonenumber.js';`

**Comment by janmarek** [comment]
Yes, but that's another problem. The component is supposed to send that request every time it has a utilsScript prop, isn't it? Why would it have that prop otherwise?

**Comment by patw0929** [comment]
I copied your demo code and trying to make a demo again.

Using `react-intl-tel-input` v2.0.12 here.

`src/app.js`:

```javascript
import React from 'react';
import ReactDOM from 'react-dom';
import PhoneInput from './input';

class Demo extends React.Component {
  render() {
    return (
      <PhoneInput />
    );
  }
}

ReactDOM.render(<Demo />, document.getElementById('content'));
```

`src/input.js`:

```javascript
import React from 'react';
import IntlTelInput from 'react-intl-tel-input';
import classnames from 'classnames';

import 'file?name=libphonenumber.js!../node_modules/react-intl-tel-input/dist/libphonenumber.js';
import '../node_modules/react-intl-tel-input/dist/main.css';


export default class PhoneInput extends React.Component {
  static defaultProps = {
    label: 'abc',
    labelClassName: 'labelClass',
  };

  render() {
    return <div className="form-group">
      {this.props.label && <label className={classnames('control-label', this.props.labelClassName)}>
        {this.props.label}
      </label>}
      <IntlTelInput
        css={['intl-tel-input', 'form-control']}
        value={'0999 000 001'}
        utilsScript="./libphonenumber.js"
      />
    </div>;
  }
}
```

Result:

![2016-03-01 6 55 37](https://cloud.githubusercontent.com/assets/3055294/13425022/38dfe3b2-dfdf-11e5-9117-55024551d76a.png)

There is a request about `libphonenumber.js`:

![2016-03-01 6 53 58](https://cloud.githubusercontent.com/assets/3055294/13424990/02969274-dfdf-11e5-9822-e6f32a0798ea.png)


It seems works. Any ideas?

**Comment by jackfengji** [comment]
I think it's not working when the component is only mounted after page has been loaded (for example: by a click event)

**Comment by stevenp** [comment]
This is because the way that `windowLoaded` works doesn't take into account whether the window is already loaded. I'm going to fix it and open a PR.

**Comment by stevenp** [comment]
Fixed in #81.

**Comment by patw0929** [comment]
Thanks for help! Fixed in v2.2.3.
